modules/sdk-coin-sol/src/lib/constants.ts

```diff
@@ -114,3 +114,188 @@ export const ataInitInstructionIndexes = {
 
 export const nonceAdvanceInstruction = 'AdvanceNonceAccount';
 export const validInstructionData = '0a00000001000000';
+
+export const SOL_MAINNET_TOKENS = {
```

Comment:
Shouldn't this live in statics instead? 

https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts